bouncer/app.py

```diff
@@ -45,7 +45,7 @@ def settings():
     return result
 
 
-def app():
+def create_app(_=None, **_settings):
```

Comment:
Renamed `app` to `create_app()` like our other apps. Bouncer was also passing Gunicorn a WSGI app rather than a WSGI app factory: the `gunicorn` command was `gunicorn bouncer.app:app()` rather than `gunicorn bouncer.app:app`. I've changed it to the factory version the same as our other apps, hence why the function needs to have arguments now although it doesn't actually pay any attention to them.